app/src/ui-blokada/kotlin/core/bits/Slots.kt

```diff
@@ -38,6 +38,7 @@ class FiltersStatusVB(
 
     private var rules: Int = 0
     private var memory: Int = 0
+    private var maxMemory = getMaxMemory()
```

Comment:
This variable is not needed. Just move the function call to where you use the var. :)

Reply:
Thanks for the feedback.
The variable is removed and the function is called in line 85, changes are committed and pushed.